index.js

```diff
@@ -1,54 +1,44 @@
 const exec = require('child_process').exec
 const createDebugger = require('debug')
 const path = require('path')
+const _ = require('underscore')
 
 const depsCommand = `npm ls --parseable`
 
-function splitOnNewLine(string) {
-  let dbgr = createDebugger(arguments.callee.name)
-  //dbgr(string)
+function mapOutputToDepList(string) {
   return string.split('\n')
 }
 
 function pathOfPackage(string) {
-  // pass dependency chain with
-  let dbgr = createDebugger(arguments.callee.name)
-  //dbgr(string)
   return path.join(string, "package.json")
 }
 
 function packageFor(string) {
-  let dbgr = createDebugger(arguments.callee.name)
-  //dbgr(string)
   return require(string)
 }
 
-function keyFromPackage(key) {
+function keysFromPackage(keys) {
   return object => {
-    let dbgr = createDebugger(arguments.callee.name)
-    //dbgr(JSON.stringify(object))
-
-    return { packageName: object.name, version: object.version, keyValue: object[key] }
+    return _.pick(object, keys)
   }
 }
 
 function tailOfPath(string) {
-  let dbgr = createDebugger(arguments.callee.name)
-  //dbgr(string)
   return string.split('/').pop()
 }
 
 function removeEmptyStrings(item) {
   return item !== ''
 }
 
-function start(key, cb) {
-  let dbgr = createDebugger(arguments.callee.name)
-  dbgr("In start function")
+Array.prototype.removeThisPackage = function() {
```

Comment:
I feel uneasy with modifying the `Array.prototype`.  If this library were always run by itself, that would not be an issue, but this could become part of a running server, in which case that modification becomes part of that app too.

Since you're already pulling in `underscore`, what do you think of: [http://underscorejs.org/#rest](http://underscorejs.org/#rest).  It is also aliased as `_.tail`.  

Or, if you prefer `lodash`: https://lodash.com/docs#tail